packages/gatsby-cli/README.md

```diff
@@ -32,10 +32,47 @@ You can also use the `package.json` script variant of these commands, typically
 
 ### `new`
 
-`gatsby new gatsby-site`
+```bash
+gatsby new [<site-name> [<starter-url>]]
+```
+
+#### Arguments
+
+- **`site-name` :** Your Gatsby site name along with the path should be pass here`
+
+- **`starter-url` :** Mention the gatsby starter pack url or local path here.
+  See the [Gatsby starters](https://www.gatsbyjs.org/docs/gatsby-starters/) for more.
+  `Default` starter pack is [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default).
+
+> **NOTE :** Dont pass `.` or `./` or `` _(space)_ in the `site-name`. It will throw error of specifying name and it will stop the operation
```

Comment:
I feel like full-on sentences are bit easier to read.

```suggestion
> Note: The `site-name` should only consist of letters and numbers. If you specify a `.`, `./` or a `<space>` in the name, `gatsby new` will throw an error.
```

Reply:
Nice 👍